commands/command_checkout.go

```diff
@@ -107,14 +107,30 @@ func checkoutWithIncludeExclude(include []string, exclude []string) {
 		checkoutWithChan(c)
 		wait.Done()
 	}()
+
+	// Count bytes for progress
+	var totalBytes int64
+	for _, pointer := range pointers {
+		totalBytes += pointer.Size
+	}
+	progress := lfs.NewProgressMeter(len(pointers), totalBytes, false)
```

Comment:
Ah OK, Start() wasn't there until I rebased, done now. You have to use quite big files to see the progress otherwise most of the time (1-2s) is spent at the end waiting for the update-index process to stop.